Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml

```diff
@@ -0,0 +1,20 @@
+- type: entity
+  parent: LockerChiefJustice
+  id: LockerChiefJusticeFilled
+  name: chief justice's locker
```

Comment:
![image](https://github.com/DeltaV-Station/Delta-v/assets/113523727/36fe2dde-c132-4f5c-b338-523b40b209a0)

this shares the same name as the empty one without any suffix, making it confusing for mappers